src/trace_processor/importers/systrace/systrace_line_parser.cc

```diff
@@ -149,6 +149,44 @@ base::Status SystraceLineParser::ParseLine(const SystraceLine& line) {
     TrackId track = context_->track_tracker->InternTrack(
         tracks::kCpuFrequencyBlueprint, tracks::Dimensions(event_cpu.value()));
     context_->event_tracker->PushCounter(line.ts, new_state.value(), track);
+  } else if (line.event_name == "cpu_frequency_limits") {
+    std::optional<uint32_t> event_cpu = base::StringToUInt32(args["cpu_id"]);
+    if (!event_cpu.has_value()) {
+      return base::Status("Could not convert event cpu");
+    }
+    std::optional<double> max_freq =
+        args.Find("max") != nullptr ? base::StringToDouble(args["max"])
+                                    : base::StringToDouble(args["max_freq"]);
+    std::optional<double> min_freq =
+        args.Find("min") != nullptr ? base::StringToDouble(args["min"])
+                                    : base::StringToDouble(args["min_freq"]);
+    if (max_freq.has_value()) {
+      static constexpr auto kMaxBlueprint = tracks::CounterBlueprint(
```

Comment:
Given these are shared between proto and systrace, can you please move these to `src/trace_processor/importers/common/tracks_common.h` and change ftrace_parser.cc to also reference the common definition?